src/server/scripts/Northrend/VaultOfArchavon/boss_toravon.cpp

```diff
@@ -12,24 +12,31 @@
 enum Spells
 {
     // Toravon
-    SPELL_FREEZING_GROUND               = 72090,
-    SPELL_FROZEN_ORB                    = 72091,
-    SPELL_WHITEOUT                      = 72034,
-    SPELL_FROZEN_MALLET                 = 71993,
+    SPELL_FREEZING_GROUND              = 72090,
+    SPELL_FROZEN_ORB                   = 72091,
+    SPELL_WHITEOUT                     = 72034,
+    SPELL_WHITEOUT_VISUAL              = 72036,
+    SPELL_FROZEN_MALLET                = 71993,
 
     // Frozen Orb
     SPELL_FROZEN_ORB_DMG                = 72081,    // priodic dmg aura
     SPELL_FROZEN_ORB_AURA               = 72067,    // make visible
 
     // Frozen Orb Stalker
     SPELL_FROZEN_ORB_STALKER_VISUAL     = 72094,
+
+    // Whiteout GroundEffect NPC
+    NPC_WHITEOUT_GROUND_EFFECT          = 500005,
```

Comment:
Now where sql this npc ? i don't find entry 500005 in DB.

Reply:
> Now where sql this npc ? i don't find entry 500005 in DB.

a sql file is attached on this PR

Reply:
How to i apply your suggestions ? no "apply suggestion" button shows....

Reply:
the 500005 creature
data/sql/updates/pending_db_world/rev_1586524859560115300.sql

Reply:
i don't see any sql :

https://github.com/azerothcore/azerothcore-wotlk/pull/2868/files

Reply:
i find , you delete sql file on this commit:
https://github.com/azerothcore/azerothcore-wotlk/pull/2868/commits/cfe342c6a7ec57ba86ccb0dbbcf225120b999d73

again must pull new commit

Reply:
WTF, sorry i new on github.... some misclick

Reply:
commited

Reply:
You know some way to generate a best and friendly ID for the creature ? instead of 500005 ?

Reply:
> You know some way to generate a best and friendly ID for the creature ? instead of 500005 ?

You can use an entry near boss entry, if not in WowHead.

Reply:
Just check so you dont override another creature when you check for a new entry :)

Reply:
Entry updated to math boss range